app/src/main/java/com/codingblocks/cbonlineapp/mycourse/content/codechallenge/CodeChallengeViewModel.kt

```diff
@@ -24,6 +24,11 @@ class CodeChallengeViewModel(
     var codeChallenge: CodeChallenge? = null
 
     fun fetchCodeChallenge() = liveData(Dispatchers.IO) {
+        if (codeId.isNullOrEmpty()){
```

Comment:
Wouldn't it be empty every time ?🤔

Reply:
It would not be empty when called from Courses list. At that moment we have the codeID and contestId.

Reply:
That has to be changed i mean there would be only way to open the content that would be by passing section and content id

Reply:
Okay will change that